src/api/common/EntityFunctions.ts

```diff
@@ -59,7 +59,7 @@ export const modelInfos = {
 } as const
 export type ModelInfos = typeof modelInfos
 
-export async function resolveTypeReference(typeRef: TypeRef<any>): Promise<TypeModel> {
+export function resolveTypeReference(typeRef: TypeRef<any>): TypeModel {
```

Comment:
the idea was to *not* bundle all the TypeModels into JS files as they are big and slow to process and instead load them as JSON, that's why this one is async.

IDK if it will matter with the SDK though, we embed things into binary there because it's not JS and doesn't cost nearly as much

Reply:
Does this mean we should undo the change? If so we should document this requirement in the code.

Reply:
I would roll it back for now, it gives us more flexibility in the future. It should have been documented, not faulting you here

Reply:
Ok, just changed it.

Reply:
thank you!